cvat/apps/engine/mixins.py

```diff
@@ -279,8 +279,7 @@ def export_annotations(self, request, db_obj, export_func, callback, get_data=No
 
         data = get_data(self._object.pk)
         serializer = LabeledDataSerializer(data=data)
-        if serializer.is_valid(raise_exception=True):
-            return Response(serializer.data)
+        return Response(serializer.initial_data)
```

Comment:
``get_data`` is passed as an argument

It is a minor problem because in general it does not guarantee us that it receives "safe" data from the database..